<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDPFileTransferWindow</class>
 <widget class="QMainWindow" name="UDPFileTransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDPFileTransferWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
background-image:none;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLineEdit" name="srcPNLE">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;
border-bottom :2px solid black;
font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dstIPLE">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;
border-bottom : 2px solid black;
font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="dstPNLE">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none;
border-bottom : 2px solid black;
font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="srcPNLbl">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
    <property name="text">
     <string>Source Port Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="dstIPLbl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
    <property name="text">
     <string>Destination IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="dstPNLbl">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Tahoma&quot;;
</string>
    </property>
    <property name="text">
     <string>Destination Port Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectPB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Tahoma&quot;;
</string>
    </property>
    <property name="text">
     <string>   Connect</string>
    </property>
    <property name="icon">
     <iconset resource="images/designicons.qrc">
      <normaloff>:/icons/signal.png</normaloff>:/icons/signal.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendFilePB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images/designicons.qrc">
      <normaloff>:/icons/file64.png</normaloff>:/icons/file64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>298</number>
    </property>
    <property name="autoRepeatInterval">
     <number>99</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="injectErrorsCB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inject Errors</string>
    </property>
   </widget>
   <widget class="QLabel" name="seperatorLbl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left:2px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loadingLbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/designicons.qrc"/>
 </resources>
 <connections/>
</ui>
